index.js

```diff
@@ -7,16 +7,16 @@ const LINE_OFFSET = 7
 const { symbols } = require('pino')
 const { asJsonSym } = symbols
 
-function traceCaller (pinoInstance, relativeTo) {
+function traceCaller (pinoInstance, options = {relativeTo: null}) {
   function get (target, name) {
     return name === asJsonSym ? asJson : target[name]
   }
 
   function asJson (...args) {
     args[0] = args[0] || Object.create(null)
     args[0].caller = Error().stack.split('\n').slice(2).filter(s => !s.includes('node_modules/pino') && !s.includes('node_modules\\pino'))[STACKTRACE_OFFSET].substr(LINE_OFFSET)
-    if (typeof relativeTo === 'string') {
-      args[0].caller = args[0].caller.replace(relativeTo + '/', '').replace(relativeTo + '\\', '')
+    if (typeof options === 'object' && options !== null && typeof options.relativeTo === 'string') {
```

Comment:
i think it's safe to assume that `options` will always be a defined symbol so that the existing check can be simplified as the following:

```suggestion
    if (options && typeof options.relativeTo === 'string') {
```